symphony/lib/boot/defines.php

```diff
@@ -211,11 +211,32 @@
 define_safe('HTTPS', server_safe('HTTPS'));
 
 /**
- * Returns the current host, ie. google.com
+ * Returns the current host, e.g. google.com
```

Comment:
Out of curiosity, why not `i.e.` ?

Reply:
AFAIK i.e. means "that is", e.g. means "for example". The host is not always google.com, it's just an example. (We might es well use example.com here.)

Reply:
Cool, thanks for educating me.